client/src/components/Dataset/DatasetDisplay.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { BAlert } from "bootstrap-vue";
+import { storeToRefs } from "pinia";
+import { computed, ref, watch } from "vue";
+
+import { useDatasetStore } from "@/stores/datasetStore";
+import { useUserStore } from "@/stores/userStore";
+import { withPrefix } from "@/utils/redirect";
+import { errorMessageAsString } from "@/utils/simple-error";
+import { bytesToString } from "@/utils/utils";
+
+import Alert from "@/components/Alert.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+import CenterFrame from "@/entry/analysis/modules/CenterFrame.vue";
+import TabularChunkedView from "components/Dataset/Tabular/TabularChunkedView.vue";
```

Comment:
```suggestion
import TabularChunkedView from "@/components/Dataset/Tabular/TabularChunkedView.vue";
```
You will need to reorder the imports after this change.